### ISSUE TITLE
no effect at all

### ISSUE THREAD

**Issue by plsharevme** [onlyxuyang/javsitemarker#1]
after key in jellyfin api and address , nothing happen .....it still useful?
thanks

**Comment by onlyxuyang** [comment]
yes, I think it is useful. Are movies in jellyfin has right Code(eg:PPPD-806) in name？ This plugin query code found in website and using jellyfin query interface to found the code in jellyfin. This need your movies has right labels in jellyfin. Moive name must be its code.
